package.json

```diff
@@ -41,6 +42,7 @@
     "@supabase/supabase-js": "^2.39.3",
     "@tanstack/react-query": "^5.17.15",
     "@tanstack/react-query-devtools": "^5.17.15",
+    "@tanstack/react-table": "^8.20.5",
```

Comment:
Where did you use this?

Reply:
@Simpcyclassy so basically shadcn need to integrate this to work